.github/workflows/build.yml

```diff
@@ -0,0 +1,32 @@
+name: Build and verify PR Title
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  verify-pr-title:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
```

Comment:
This can be removed as well for the same reason mentioned below

Reply:
removed